### PR TITLE
Azbl/875 Fix and hyperlink PA horticultural society on homepage.

### PR DESCRIPTION
**Pull request by AZBL**
This request:

1. Changes Pennsylvania Horticulture Society to Pennsylvania Horticultural Society on homepage.
2. Creates a hyperlink for the text Pennsylvania Horticultural Society to https://phsonline.org/

### CODE REVIEW

**Comment by AZBL** [comment]
@bacitracin @CodeWritingCow

File: src/app/(content-pages)/page.tsx [hunk 1]
```diff
@@ -139,7 +139,22 @@ const LandingPage: FC = () => {
...
+              <>
+                Community groups have been cleaning up lots in their own
+                neighborhoods for decades. Large organizations like the
+                <a
+                  target="_blank"
+                  rel="noopener noreferrer nofollow"
+                  href="https://phsonline.org/"
+                  className="default"
+                >
+                  Pennsylvania Horticultural Society
```
Comment (CodeWritingCow) [inline src/app/(content-pages)/page.tsx:152]: Add space between "the" and "Pennsylvania."

Screen Shot 2024-09-23 at 11 44 00 PM

**Review by CodeWritingCow — CHANGES_REQUESTED** [review]
Hi @AZBL, the society's name has been corrected, and the new link opened correctly. Just add a space between "the" and "Pennsylvania," and I'll approve this PR. Thanks!

**Comment by AZBL** [comment]
@CodeWritingCow I added the space so hopefully it's good to go.

**Pull request merged**